themes/navy/layout/partial/sidebar.swig

```diff
@@ -1,5 +1,30 @@
 <aside id="sidebar" role="navigation">
   <div class="inner">
+    <label id="version-select-container"><b>Version:</b> <select id="version-select">
+        <option value="3.2">3.2.x</option>
+        <option value="4.alpha">4.alpha.x</option>
+      </select></label>
+    <script>
+      const versionSelector = document.getElementById('version-select');
+      // TODO change these URLS to the right ones once Iuri or someone else has create the Netlify project
+      // These still work right now however
+      // TODO use a config file?
+      const VERSIONS = {'3.2': 'https://embark.status.im/docs', '4.alpha': 'https://embark-site-develop.netlify.com/docs/'};
```

Comment:
When/how do we update this with `4.beta.x` ? If we put this change out, it should include versions for beta.x as well, no?

Reply:
Oh, oops, forgot to change it to beta. That's the only part that needs manual work. Maybe we can put this later in a config file that is shared between "versions"